doc/source/conf.py

```diff
@@ -13,7 +13,8 @@
 from ansys_sphinx_theme import (
     __version__,
     ansys_favicon,
-    ansys_logo_black,
+    ansys_logo_dark_mode,
+    ansys_logo_light_mode,
     ansys_logo_white,
```

Comment:
Can we avoid breaking changes and add a deprecation warning to whomever is using the old images?

Reply:
Sure, we can add docs and depreciation warning, i will add those in another PR. i thought of discuss it on our meeting (about logos and upcoming changes with sidebar)